source/core/parameters_multiphysics.cc

```diff
@@ -68,6 +68,13 @@ Parameters::Multiphysics::declare_parameters(ParameterHandler &prm)
                       "false",
                       Patterns::Bool(),
                       "Buoyant force calculation <true|false>");
+
+    // subparameter for tracer
+    prm.declare_entry(
+      "first order reaction",
+      "false",
+      Patterns::Bool(),
+      "First order reaction calculation for tracer physics <true|false>");
```

Comment:
If you want to be really general, maybe I would go for an int giving the order of the reaction itself. Like this, it would work for any order (0 to infiniiiiity :D )

Reply:
Done!